providers/source.rb

```diff
@@ -40,7 +40,7 @@
     end
 
     variables(type: new_resource.type,
-              parameters: parameters,
+              parameters: TdAgent::Helpers.params_to_text(parameters),
```

Comment:
This must be the cause of CI failure. The ERB template for `source` is expecting `parameters` as a `Hash`, but yours changed the parameter type to `String`.

https://github.com/treasure-data/chef-td-agent/pull/108/files#diff-f147331bc276bd1ee4f347f715012f0aR13

I think you need to update `templates/default/source.conf.erb` to handle `@parameters` as a `String`